saleor/graphql/core/utils/__init__.py

```diff
@@ -142,15 +144,30 @@ def validate_required_string_field(cleaned_input, field_name: str):
 def from_global_id_or_error(
     id: str, only_type: Union[ObjectType, str] = None, raise_error: bool = False
 ):
-    """Resolve database ID from global ID or raise ValidationError.
+    """Resolve global ID or raise GraphQLError.
 
+    Validates if given ID is a proper ID handled by Saleor.
+    Valid IDs formats, base64 encoded:
+    'app:<int>:<str>' : External app ID with 'app' prefix
+    '<type>:<int>' : Internal ID containing object type and ID as integer
+    '<type>:<UUID>' : Internal ID containing object type and UUID
     Optionally validate the object type, if `only_type` is provided,
     raise GraphQLError when `raise_error` is set to True.
     """
     try:
         _type, _id = graphene.Node.from_global_id(id)
```

Comment:
I don't think it's mentioned by PEP-8 but generally any local variable starting by `_` indicates it's not being used while giving a useful description to what it contains.

`_type` and `_id` should rather be `type_` and `id_` (or `idx`)

Reply:
fixed